client-app/pages/cart.vue

```diff
@@ -113,7 +113,7 @@
 
         <!-- Create quote widget -->
         <VcWidget
```

Comment:
Can we remove that widget and add it from the module by some kind of "extension point" with vue teleport?

Reply:
Show me how :)